README.md

```diff
@@ -163,3 +163,12 @@ This comparison describes only free tiers of these services.
 ]
 ```
 </details>
+
+## Zeabur.com
+
+[Pricing](https://zeabur.com/pricing) | [Deployment](https://docs.zeabur.com/get-started)
+
+* Has Postgres, Redis, MongoDB, MySQL and other databases
+* Include a US$ 5 free credits every month
```

Comment:
I think it would be fair to add such line similarly to Railway:

```md
* Can't run an app 24/7 at a free tier because it offers only 500 hours of usage per month
```

```suggestion
* Can't run an app 24/7 at a free tier, include a US$ 5 free credits every month
```

Reply:
But if your service doesn't use too many resources, Zeabur can run an app 24/7 at a free tier.
For example, a service with 100MB in memory and 0.2v CPU

Reply:
yes this is much better :)
thanks for review!